### PR TITLE
refactor(unique): move to helpers

### PR DESCRIPTION
**Pull request by xDivisionByZerox**
This PR moves the `unique()` method from the `Unique` module into the `Helpers` module. There is no reason for it being a standalone module when it is clearly a utility method.

This was discussed in https://github.com/faker-js/faker/discussions/805 ([direct link](https://github.com/faker-js/faker/discussions/805#discussioncomment-2539551)).

### CODE REVIEW

File: src/modules/unique/index.ts [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-import * as uniqueExec from './unique';
 
 /**
  * Module to generate unique entries.
+ *
+ * @deprecated
  */
 export class Unique {
-  constructor() {
+  constructor(private readonly faker: Faker) {
```
Comment (xDivisionByZerox) [inline src/modules/unique/index.ts:11]: Does this change make the entire PR "breaking"?

Comment (Shinigami92) [reply src/modules/unique/index.ts:11]: We could be in luck here, as we do not yet export the Module constructors on their own

File: src/modules/helpers/index.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import type { Faker } from '../..';
 import { FakerError } from '../../errors/faker-error';
+import type { RecordKey } from '../unique/unique';
+import * as uniqueExec from '../unique/unique';
```
Comment (ST-DDT) [inline src/modules/helpers/index.ts:4]: IMO these should be moved the helpers module (folder) too.

File: src/modules/unique/index.ts [hunk 3]
```diff
@@ -52,13 +59,12 @@ export class Unique {
...
-    const { maxTime = 50, maxRetries = 50 } = options;
-    return uniqueExec.exec(method, args, {
-      ...options,
-      startTime: new Date().getTime(),
-      maxTime,
-      maxRetries,
-      currentIterations: 0,
+    deprecated({
+      deprecated: 'faker.fake()',
+      proposed: 'faker.helpers.fake()',
```
Comment (Shinigami92) [inline src/modules/unique/index.ts:64]: `fake`?! what? I thought this is `unique` 😆

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

File: src/modules/unique/index.ts (hunk 3 above)
Comment (xDivisionByZerox) [reply src/modules/unique/index.ts]: Done. Was copying the deprecation from `fake` for convenience reasons^^

**Review by Shinigami92 — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**